bin/cml/pr.js

```diff
@@ -39,6 +39,24 @@ exports.builder = (yargs) =>
         conflicts: ['merge', 'rebase'],
         description: 'Try to squash-merge the pull request upon creation.'
       },
+      branch: {
+        type: 'string',
+        description: 'Set a custom branch name for pull request.'
+      },
+      title: {
+        type: 'string',
+        description: 'Title of the created pull request.'
+      },
+      body: {
+        type: 'string',
+        description:
+          'Description/body of created pull request [string or file]. Ex "My cml pr", "reports/experiment-results.md"'
+      },
+      message: {
+        type: 'string',
+        description:
+          'The commit message for the new commit used to open the pull request'
```

Comment:
```suggestion
          'Commit message for the commit used to open the pull request'
```

Surely @casperdcl can find something even better. 💅🏼